src/pages/index.js

```diff
@@ -5,96 +5,235 @@ import PopupWithImage from "../components/PopupWithImage.js";
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "a5df8bb7-ccf7-4af1-a820-819df810a6c4",
+    "Content-Type": "application/json",
+  },
+});
 
+Promise.all([api.getInitialCards(), api.getUserInfo()])
+  .then(([cards, userInfo]) => {
+    console.log(cards, userInfo);
+  })
+  .catch((err) => console.error(err));
+
+let cardSection;
+
+api
+  .getInitialCards()
+  .then((cards) => {
+    cardSection = new Section(
+      {
+        items: cards,
+        renderer: (data) => {
+          const cardElement = createCard(data);
+          cardSection.addItem(cardElement);
+        },
+      },
+      ".cards__list"
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+//initializing user info
 const userInfo = new UserInfo({
   nameSelector: "#profile-title",
   descriptionSelector: "#profile-job",
+  avatarImage: "#profile-image",
 });
 
-//Initializing add place popup window
+api.getUserInfo().then((userData) => {
+  userInfo.setUserInfo(userData.name, userData.about);
+  userInfo.setUserAvatar(userData.avatar);
+});
+
+//Initializing popup windows
 const addPlacePopup = new PopupWithForm(
   "#add-place-form",
   handleAddCardFormSubmit
 );
 
-addPlacePopup.setEventListeners();
-
-//Initializing add place popup window
+//Initializing profile popup window
 const profileEditPopup = new PopupWithForm(
   "#profile-edit-window",
   handleProfileFormSubmit
 );
-profileEditPopup.setEventListeners();
+
+const editAvatarPopup = new PopupWithForm(
+  "#edit-avatar",
+  handleAvatarFormSubmit
+);
 
 const cardPreview = new PopupWithImage("#image-popup", handleImageClick);
 
-//Edit profile window consts
-const profileFormElement = document.forms["profile-edit-fields"];
+const deletePopup = new DeletePopup("#delete-card", deleteCardForm);
+
+//Set eventlisteners
+
+addPlacePopup.setEventListeners();
+profileEditPopup.setEventListeners();
+editAvatarPopup.setEventListeners();
+deletePopup.setEventListeners();
+
+//Form Validaors
 
 const editFormValidator = new FormValidator(config, profileFormElement);
 editFormValidator.enableValidation();
 
-//New card window consts
-const addNewPlaceForm = document.forms["add-place-edit-fields"];
-
 const addFormValidator = new FormValidator(config, addNewPlaceForm);
 addFormValidator.enableValidation();
 
+const avatarFormValidator = new FormValidator(config, editAvatarForm);
+avatarFormValidator.enableValidation();
+
+//Open Modals
+
 function handleImageClick(data) {
   cardPreview.open(data);
 }
 
-//Button consts
-const profileAddButton = document.querySelector(".profile__add-button");
-const profileEditButton = document.querySelector(".profile__edit-button");
-
-// button event listeners
-
 profileAddButton.addEventListener("click", () => {
   addPlacePopup.open();
 });
 
+editAvatarButton.addEventListener("click", () => {
+  editAvatarPopup.open();
+  avatarFormValidator.disableButton();
```

Comment:
`avatarFormValidator.disableButton();` should be in block `then` 

You need to `disable` the button only after **resetting** the form when you have submitted a card/avatar otherwise users may close the modal by accident and they don’t expect that the button will be disabled on opening if the inputs have **valid** data which they haven’t submitted yet.